build-tools/packages/build-cli/src/library/repoPolicyCheck/fluidBuildTasks.ts

```diff
@@ -221,16 +222,12 @@ function eslintGetScriptDependencies(
 	let config: EslintConfig;
 	try {
 		const { ext } = path.parse(eslintConfig);
-		if (ext !== ".js" && ext !== ".cjs") {
+		if (ext !== ".js" && ext !== ".cjs" && ext !== ".mjs") {
```

Comment:
Might just be better to check if the extension is json.

Reply:
Extension won't always be .json. Could be nothing (`.eslintrc`) and be expected as json.